native/python/pyjp_value.cpp

```diff
@@ -140,10 +140,12 @@ void PyJPValue_finalize(void* obj)
 	JPValue* value = PyJPValue_getJavaSlot((PyObject*) obj);
 	if (value == NULL)
 		return;
+	JPContext *context = JPContext_global;
+	if (context == NULL || !context->isRunning())
+		return;
```

Comment:
If the JVM is gone we don't need to dereference, so no need to to proceed further.